Data/uomReferenceData-starter-set.ttl

```diff
@@ -0,0 +1,2650 @@
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_activity> <http://www.w3.org/2000/01/rdf-schema#seeAlso> "http://en.wikipedia.org/wiki/Mass_number"^^<http://www.w3.org/2001/XMLSchema#anyURI> .
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_activity> <http://www.w3.org/2000/01/rdf-schema#seeAlso> "http://en.wikipedia.org/wiki/Radioactive_decay"^^<http://www.w3.org/2001/XMLSchema#anyURI> .
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_activity> <http://www.w3.org/2000/01/rdf-schema#seeAlso> "http://en.wikipedia.org/wiki/Radioactive_decay#Radioactive_decay_rates"^^<http://www.w3.org/2001/XMLSchema#anyURI> .
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_activity> <http://www.w3.org/2000/01/rdf-schema#seeAlso> "http://www.iso.org/iso/catalogue_detail?csnumber=31895"^^<http://www.w3.org/2001/XMLSchema#anyURI> .
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_activity> <http://www.w3.org/2004/02/skos/core#closeMatch> <http://qudt.org/vocab/quantitykind/Activity> .
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_activity> <http://www.w3.org/2004/02/skos/core#definition> "from QUDT: \"Activity\" is the number of decays per unit time of a radioactive sample, the term used to characterise the number of nuclei which disintegrate in a radioactive substance per unit time. Activity is usually measured in Becquerels (\\(Bq\\)), where 1 \\(Bq\\) is 1 disintegration per second, in honor of the scientist Henri Becquerel." .
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_activity> <http://www.w3.org/2004/02/skos/core#prefLabel> "activity" .
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_activity> <https://w3id.org/semanticarts/ns/ontology/gist/hasUnitGroup> <https://w3id.org/semanticarts/ns/data/gist/_UnitGroup_activity> .
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_activity> <https://w3id.org/semanticarts/ns/ontology/gist/isCategorizedBy> <https://w3id.org/semanticarts/ns/data/gist/_Discipline_radiology> .
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_altitude> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://w3id.org/semanticarts/ns/ontology/gist/Aspect> .
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_altitude> <http://www.w3.org/2004/02/skos/core#closeMatch> <http://qudt.org/vocab/quantitykind/Altitude> .
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_altitude> <http://www.w3.org/2004/02/skos/core#definition> "The aspect altitude."^^<http://www.w3.org/2001/XMLSchema#string> .
+<https://w3id.org/semanticarts/ns/data/gist/_Aspect_altitude> <http://www.w3.org/2004/02/skos/core#definition> "from QUDT: Altitude or height is defined based on the context in which it is used (aviation, geometry, geographical survey, sport, and more). As a general definition, altitude is a distance measurement, usually in the vertical or \"up\" direction, between a reference datum and a point or object. The reference datum also often varies according to the context. [Wikipedia]" .
```

Comment:
Which parts here are from QUDT, and which from Wikipedia? Or did QUDT get their definition from Wikipedia?